package.json

```diff
@@ -96,7 +99,6 @@
     "@types/randombytes": "^2.0.1",
     "@types/sinon": "^17.0.2",
     "@types/urijs": "^1.19.20",
-    "@typescript-eslint/parser": "^6.14.0",
```

Comment:
why gone?

Reply:
Oversight. I thought I added this in an early commit and then removed it. I'll add it back.